spec/requests/api/v1/documents_spec.rb

```diff
@@ -310,6 +319,39 @@
 
       it_behaves_like 'a data modifying request made during read-only mode'
     end
+
+    context 'when time has passed since the document was created' do
```

Comment:
I'm deliberately adding this as an integration spec. Currently this functionality come courtesy of a gem that we're about to remove, so I feel more confident testing this at the integration level, so that the spec is not dependent on any low-level implementation.